cmd/containerd/main_linux.go

```diff
@@ -33,12 +34,6 @@ func platformInit(context *cli.Context) error {
 			return err
 		}
 	}
-	if err := os.MkdirAll(conf.State, 0750); err != nil {
```

Comment:
Should have put my comment on the lower line. Yes it is creating the directory, but is creating it without `g+x` and no `chown`.